mysql/README.md

```diff
@@ -0,0 +1,233 @@
+## MySQL Example Usage
+
+The following examples assume `harbourbridge` has been added to your PATH
+environment variable.
+
+To use HarbourBridge on a MySQL database called mydb using mysqldump, run:
```

Comment:
I think it might be worth highlighting the mysqldump vs direct method a bit more. In the current text it is kind of lost at the bottom of the page. How about:

1) Inserting a sentence at the start of the paragraph something like: "HarbourBridge can either be used with mysqldump or it can be run directly on a MySQL database (via go's database/sql package)."
2) Add a sub-heading: "Using HarbourBridge with mysqldump"
3) Add a sub-heading: "Directly connecting to a MySQL database"

Reply:
Done. Updated `postgres/README.go` as well.